code/modules/events/scrubber_overflow.dm

```diff
@@ -53,7 +51,7 @@
 		/datum/reagent/consumable/ethanol/beer,
 		/datum/reagent/hair_dye,
 		/datum/reagent/consumable/sugar,
-		/datum/reagent/glitter/white,
+		/datum/reagent/glitter,
```

Comment:
I feel it'd be kind of sad if this could only ever spew out the default colour white glitter
And on the other end, it'd be *really* funny if this could potentially be a mixed glitter
Any way you could think of to make this more variable?

Reply:
I'm not sure whether this is compatible with reagent data at all, if so it's easy, otherwise oh god no

Reply:
I wonder how hard it would be to default to white when created through the recipe, and otherwise default to anything from a list of random options
Probably annoying given the former might happen while there's already reagents in there, hrmn